pkg/services/rpcsrv/server.go

```diff
@@ -2449,21 +2446,35 @@ func getRelayResult(err error, hash util.Uint256) (any, *neorpc.Error) {
 		return result.RelayResult{
 			Hash: hash,
 		}, nil
-	case errors.Is(err, core.ErrAlreadyExists):
+	case errors.Is(err, core.ErrTxExpired):
+		return nil, neorpc.WrapErrorWithData(neorpc.ErrExpiredTransaction, err.Error())
+	case errors.Is(err, core.ErrAlreadyExists) || errors.Is(err, core.ErrHasConflicts) || errors.Is(err, core.ErrInvalidBlockIndex):
```

Comment:
Why is the `core.ErrHasConflicts` here?

Reply:
I thought that it's similar to ErrAlreadyExists, do you want me to delete it here? So it will be managed as default in switch.

Reply:
> I thought that it's similar to ErrAlreadyExists

It's not.

> do you want me to delete it here?

Yes.

> So it will be managed as default in switch.

It's OK, it was handled as default earlier, isn't it?

Reply:
ok, fixed